resdk/analysis/chip_seq.py

```diff
@@ -1,7 +1,9 @@
 """Chip Seq analysis."""
 from __future__ import absolute_import, division, print_function, unicode_literals
 
-from resdk.resources.utils import get_data_id, get_resource_collection, get_samples, is_sample
+from resdk.resources.utils import (
```

Comment:
Why have you changed this?

Reply:
No good reason. I first intended to put 'prepare_geo' functions in 'chip_seq.py' but then changed my mind. This was left behind.

Reply:
Or maybe knew I would have to put 'is_background' function to `resdk/analysis/utils.py` and prepared for it:)